ui/src/main/java/edu/wpi/grip/ui/MainWindowController.java

```diff
@@ -260,4 +277,43 @@ protected void deploy() {
     dialog.setResizable(true);
     dialog.showAndWait();
   }
+
+  @Subscribe
+  @SuppressWarnings({"PMD.UnusedPrivateMethod", "PMD.UnusedFormalParameter"})
+  private void runStopped(TimerEvent event) {
+    if (!(event.getTarget() instanceof PipelineRunner)) {
```

Comment:
`instanceof` is bad practice. What are you trying to do here?
Can you do something different here?